storybook/src/foundation/design-tokens/links.docs.mdx

```diff
@@ -0,0 +1,56 @@
+{/* @license CC0-1.0 */}
+
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Brand/Design Tokens/Links" />
+
+# Links
+
+We have defined a group of link styles for all link components.
+
+This includes a common set of styles,
+along with specific styles for ‘regular’ and ‘subtle’ links.
+Links can also have a contrast colour for light (but not white) backgrounds
+or an inverse colour for dark backgrounds.
+
+## Default
+
+Our default links have the following styles:
+
+| CSS variable                                               | Value                                | Description                                                                       |
+| :--------------------------------------------------------- | :----------------------------------- | :-------------------------------------------------------------------------------- |
+| `var(--ams-links-color)`                                   | `var(--ams-color-interactive)`       | The default colour for links is our interactive colour.                           |
+| `var(--ams-links-text-decoration-thickness)`               | `var(--ams-border-width-md)`         | Link underlines are our medium border width.                                      |
+| `var(--ams-links-text-underline-offset)`                   | `0.3333em`                           |                                                                                   |
+| `var(--ams-links-hover-color)`                             | `var(--ams-color-interactive-hover)` | The hover colour is our interactive hover colour.                                 |
+| `var(--ams-links-regular-hover-text-decoration-thickness)` | `var(--ams-border-width-lg)`         | The underline gets thicker on hover.                                              |
+| `var(--ams-links-regular-hover-text-underline-offset)`     | `0.2778em`                           | The offset decreases when you hover, making the underline look like it is rising. |
```

Comment:
Two occurrences of ‘regular’ here. Can’t suggest because the table layout will change.